source/reference/limitations.txt

```diff
@@ -61,6 +61,8 @@ General Limitations
   <https://www.mongodb.com/docs/atlas/atlas-search/create-index/>`__.
 - ``mongosync`` only supports clusters that use the :ref:`WiredTiger 
   <storage-wiredtiger>` storage engine.
+- ``mongosync`` does not support migrating documents that have an empty 
+  top-level timestamp, such as ``Timestamp(0,0)``.
```

Comment:
**Question** 
When you say not supported, what does this mean? Will the documents in a collection be skipped if they contain these types of documents? Will the sync completely fail? I think we should elaborate on the end result here.

Also I think adding a small example document with a json code block directive might better illustrate the types of documents that are being referenced here. 

Something like:

```
For example, the following document is not support because it has an empty timestamp field:
.. code-block:: json

   {
      "ts" : "",
      "a" : "1",
      "b" : "2"
   }

Reply:
The behavior for this is undefined  — I can explicitly mention this.

Reply:
As a follow-up to this, the top of the limitations page mentions that running `mongosync` with any of the listed limitations could lead to undefined behavior, so to avoid redundancy, I'm leaning towards leaving this as is.

Reply:
As for your second comment, I adjusted the wording to mention that _any_ document that contains an empty timestamp isn't supported. I took the copy from the [cloud docs](https://github.com/10gen/cloud-docs/pull/5360/files#diff-ac33dbc61c8f5175ac61665d5cb9f78393f0ae0254817a804ee4b215f2883572R38) regarding the same limitation.

Reply:
Awesome, I think this is much clearer for the behavior.